src/components/editor/EditAuthorsForm.jsx

```diff
@@ -84,8 +41,8 @@ export default class EditAuthorsForm extends BaseComponent {
               <div key={author.slug}>
                 {
                   !striked
-                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleDeleteAuthor.bind(this, author.slug, true)}>&times;&nbsp;</a>
-                    : <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleUnstrikeAuthor.bind(this, author.slug)}>~&nbsp;</a>
+                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.props.handleDeleteAuthor.bind(null, author.slug, true)}>&times;&nbsp;</a>
```

Comment:
By the last part of the comment about what the different browsers could do to make anchor tags feel natural, you mean that as an argument to use `<span>` right? Since then we can be more certain it shows up exactly as we want it.